core/common/src/main/java/alluxio/underfs/ObjectUnderFileSystem.java

```diff
@@ -82,6 +89,13 @@ protected ObjectUnderFileSystem(AlluxioURI uri, UnderFileSystemConfiguration ufs
     int numThreads = Configuration.getInt(PropertyKey.UNDERFS_OBJECT_STORE_SERVICE_THREADS);
     mExecutorService = ExecutorServiceFactories.fixedThreadPoolExecutorServiceFactory(
         "alluxio-underfs-object-service-worker", numThreads).create();
+
+    mOpenRetryBaseSleepMs =
+        (int) Configuration.getMs(PropertyKey.UNDERFS_OBJECT_STORE_READ_RETRY_BASE_SLEEP_MS);
+    mOpenRetryMaxSleepMs =
+        (int) Configuration.getMs(PropertyKey.UNDERFS_OBJECT_STORE_READ_RETRY_MAX_SLEEP_MS);
+    mOpenRetryMaxNum =
+        Configuration.getInt(PropertyKey.UNDERFS_OBJECT_STORE_READ_RETRY_MAX_NUM);
```

Comment:
Is the configuration call expensive? Otherwise I suggest to just do this lookup in the open call since the values will not be used anywhere else.